### DISCUSSION TITLE
How long does the "Most Used Languages" reflect upon setup?

### DISCUSSION THREAD

**Discussion by celesica** [lowlighter/metrics#1365] [Q&A] [answered]
Hello. I just recently added Metrics for my profile. I'm not sure if I did mine correctly. My workflow is below. 

![image](https://user-images.githubusercontent.com/10039521/213903727-b6e63c38-cc2c-402f-abf9-c3f585574256.png)

![image](https://user-images.githubusercontent.com/10039521/213903324-93ba56aa-5f15-408a-93a4-311179582b6f.png)

The "Recently Used" just showed up fine. It detected 13 languages, but the "Most Used" shows "No push activity found". Does it take time for them to show up? 

Thank you!

**Comment by jayvdb** [comment]
You may want to try "indepth" mode. https://github.com/lowlighter/metrics/tree/master/source/plugins/languages#-indepth-mode.

Or add private repos to the github token used by `metrics` .. ?

**Reply by celesica** [reply]
> You may want to try "indepth" mode. https://github.com/lowlighter/metrics/tree/master/source/plugins/languages#-indepth-mode.
> 
> Or add private repos to the github token used by `metrics` .. ?

Hello, the "indepth" mode is already enabled in my workflow as shown in my previous screenshot.

![image](https://user-images.githubusercontent.com/10039521/213962161-1831bb5b-8821-494c-b32e-7fdc960604e3.png)

Also for private repos, I have these already checked in my token when I set it up.

![image](https://user-images.githubusercontent.com/10039521/213962204-d54bf35e-b59c-4a4f-82c1-bf21843fc4fa.png)
![image](https://user-images.githubusercontent.com/10039521/213962238-cfaabb04-a985-4789-ab5e-fd818c877d70.png)

Hoping this gets resolved~ :(

**Reply by lowlighter** [reply]
Hi 👋 !

A bit late to answer but there's an on-going issues for private repositories (#1072). But tbh since there's some work from GitHub to allow the new fine-grained token to be used with their GraphQL API, it probably won't be looked up until then.

As for public repositories when using the `indepth` mode, it could also be linked to [`commits_authoring`](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md#commits_authoring) not being configured to match the commits signature you use (it defaults to GitHub username, but usually these are signed by emails or name)

Anyway glad you made it work in regular mode !

**Comment by celesica** [comment]
Is there any update on the Most Used Language one please? Thank you!

**Comment by celesica** [comment, marked as answer]
Solved it by removing indepth mode. Closing~

**Comment by robivictor** [comment]
In my case removing line this worked, in addition to removing indepth mode.
`plugin_languages_other: yes`
